src/main/java/org/jenkinsci/plugins/github/extension/GHEventsSubscriber.java

```diff
@@ -76,7 +77,7 @@ protected void onEvent(GHEvent event, String payload) {
         return new Function<GHEventsSubscriber, Set<GHEvent>>() {
             @Override
             public Set<GHEvent> apply(GHEventsSubscriber subscriber) {
-                return defaultIfNull(subscriber.events(), Collections.<GHEvent>emptySet());
+                return defaultIfNull(notNull(subscriber).events(), Collections.<GHEvent>emptySet());
```

Comment:
We discussed it with @lanwen and @KostyaSha . Such behavior actually follows the documentation of Guava, which suggests to throw NPEs if null values is not supported. I've originally proposed an NPE-safe implementation for all methods, but got a push-back from the reviewers